src/functions/newsletter-submission.js

```diff
@@ -0,0 +1,51 @@
+const fetch = require("node-fetch");
+const { BUTTONDOWN_API_KEY } = process.env;
+
+const config = (userEmail) => {
```

Comment:
Can we use a name that fits here a bit more? This isn't a request configuration really. I think it'd be more fitting if it was named `getRequestBody` or `buildAPIRequestBody` something that speaks to the fact that this is building the body of an HTTP request to the Buttondown API. Alternatively, we could just remove this function and call all together and put the object right in as the second parameter of `fetch`. That would be my approach here because we're only doing this once. We're not reusing the `config` function anywhere else so it's just a bit of misdirection without any strong benefits.

Reply:
I realized immediately after writing this comment that what I described doesn't make a ton of sense because this sets up the headers and the rest of the request. Sorry about that.

The idea of ditching the function still stands, but a name more like `getRequestConfig` or `buildRequestConfig` would be cool if you stick with it.

Reply:
Fixed!